cmd/libsnap-confine-private/snap.c

```diff
@@ -169,25 +169,49 @@ void sc_snap_name_validate(const char *snap_name, struct sc_error **errorp)
 	sc_error_forward(errorp, err);
 }
 
-void sc_snap_drop_instance_name(const char *snap_name, char *base,
-				size_t base_size)
+void sc_snap_drop_instance_name(const char *name, char *snap_name,
```

Comment:
Below the function is using "instance name" as the composition of a snap name and an instance key, which seems nice. But in that case we cannot say "drop instance name" here, as we'd have nothing left. This should say "drop_instance_key" instead.